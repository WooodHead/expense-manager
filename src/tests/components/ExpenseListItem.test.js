import React from 'react';
import ExpenseListItem from '../../components/ExpenseListItem';
import { shallow } from 'enzyme';
import testData from '../fixtures/expenses';

test('should output the correct expense item', () => {
    const wrapper = shallow(<ExpenseListItem key={testData[1].id} {...testData[1]} />);
    expect(wrapper).toMatchSnapshot();
});