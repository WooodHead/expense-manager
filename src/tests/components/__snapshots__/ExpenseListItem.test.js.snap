// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should output the correct expense item 1`] = `
<div>
  <Link
    replace={false}
    to="/edit/2"
  >
     
    <p>
       
      Expense 2
    </p>
  </Link>
  <p>
     
    $32.00
     - 
    December 28th 1969
  </p>
</div>
`;
